### ISSUE TITLE
u-blox nina lisa u200 and sara g350

### ISSUE THREAD

**Issue by tcpipchip** [mongoose-os-libs/pppos#25]
Lisa u200 works perfectly on mongoose pppos :)
To make the Sara works i had to comment these line! Now working perfectly too :)
![image](https://user-images.githubusercontent.com/11488460/142726184-1bab3752-322e-4dcf-af37-b2eeb9165e3d.png)

**Comment by rojer** [comment]
interesting. AT+COPS operator selection is a standard command, i don't see why it wouldn't work.
can you run at debug.level=3 and post output?

**Comment by tcpipchip** [comment]
hi, where i enable debug.level=3 ?
And do you want the debug with uncommented option, right ?

**Comment by rojer** [comment]
> hi, where i enable debug.level=3 ?

add `- ["debug.level", 3]` to your app's config_schema section or after flashing, and assuming you have rpc-uart library,
run `mos config-set debug.level=3`

> And do you want the debug with uncommented option, right ?

yes, naturally.

**Comment by tcpipchip** [comment]
attached
very weird
Attached
[capture.txt](https://github.com/mongoose-os-libs/pppos/files/7575031/capture.txt)
[capture2.txt](https://github.com/mongoose-os-libs/pppos/files/7575032/capture2.txt)
[capture3.txt](https://github.com/mongoose-os-libs/pppos/files/7575033/capture3.txt)
[capture4.txt](https://github.com/mongoose-os-libs/pppos/files/7575042/capture4.txt)
[capture5.txt](https://github.com/mongoose-os-libs/pppos/files/7575045/capture5.txt)

**Comment by tcpipchip** [comment]
i see in the source code some comments about g350, then looks that already was tested on G350.

**Comment by rojer** [comment]
that was before the quick operator reconnect was introduced, i didn't test it afterwards.

```
mgos_pppos.c:127        >> AT+COPS=0
mgos_pppos.c:137        8 -> 9 13.198152 613.194741
mgos_pppos.c:777        Command failed: AT+COPS=0
```

we don't log the response, really? dammit. can you hack the code a bit to print the response when at LL_DEBUG?

**Comment by tcpipchip** [comment]
Hi @rojer
Weird that works perfectly with SARA G450 and LISA U200.
On G350 the AT+COPS=0 returns nothing...i tested on REALTERM...very weird! Maybe is hardware G350 problem...(firmware is updated)
But if you send again AT+COPS=0 i finally gets a OK
Looks that there is not a TIME OUT implemented to AT+COPS=0 command...and stay in a LOOP.

**Comment by tcpipchip** [comment]
btw, i love this PPPos, one of more stable that i have seen!

**Comment by rojer** [comment]
indeed, weird. so maybe just implementing a retry of COPS command will do? kinda hacky, but not too difficult.

**Comment by tcpipchip** [comment]
yes, should be good! I had tried to do that...but i am still not familiar with the CODE :)

**Comment by tcpipchip** [comment]
@roger
let me know when have updates :)

**Comment by rojer** [comment]
i checked my pile of stuff and i actually happen to have the 350 in there. when i get some spare time i'll set it up and see what's up.

**Comment by tcpipchip** [comment]
Excelent!!! Excelent!!!!!!!!!!

**Comment by tcpipchip** [comment]
If works at your side @rojer , then It´s a hardware problem in my side!
